package.json

```diff
@@ -19,7 +19,7 @@
     "xml2js": "^0.4.5"
   },
   "scripts": {
-    "test": "mocha --recursive -R spec",
+    "test": "mocha --recursive -R spec --watch --reporter dot",
```

Comment:
Please create a separate script with the `--watch` option, for exmaple `test-watch`. Usually you expect that the process finished when you run `npm test`.